routeros_api/api.py

```diff
@@ -9,20 +9,20 @@
 from routeros_api import resource
 
 
-def connect(host, username='admin', password='', port=8728, force_old_login=False):
-    return RouterOsApiPool(host, username, password, port, force_old_login).get_api()
+def connect(host, username='admin', password='', port=8728, insecure_login=False):
+    return RouterOsApiPool(host, username, password, port, insecure_login).get_api()
 
 
 class RouterOsApiPool(object):
     socket_timeout = 15.
 
-    def __init__(self, host, username='admin', password='', port=8728, force_old_login=False):
+    def __init__(self, host, username='admin', password='', port=8728, insecure_login=False):
         self.host = host
         self.username = username
         self.password = password
         self.port = port
         # Don't send the new-style login (can expose password in plaintext!)
```

Comment:
You didn't flip logic in the comment and now it is misleading. I think you should just remove it.